docs/index.bs

```diff
@@ -3240,14 +3242,19 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       : Output
       :: a boolean
 
-      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"] does not [=map/exist=], return false.
-      1. Let |rawPattern| be |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"].
-      1. Let |pattern| be the result of running <a>Parse URL Pattern</a> algorithm passing |rawPattern| and |serviceWorker|. If this throws an exception, catch it and return false.
-      1. If |pattern| [=URLPattern/has regexp groups=], then return false.
+      1. Let |hasCondition| be false.
+      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"] [=map/exists=], then:
+          1. Let |rawPattern| be |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"].
+          1. Let |pattern| be the result of running <a>Parse URL Pattern</a> algorithm passing |rawPattern| and |serviceWorker|. If this throws an exception, catch it and return false.
+          1. If |pattern| [=URLPattern/has regexp groups=], then return false.
 
-          Note: Since running a user-defined regular expression has a security concern, it is prohibited.
+              Note: Since running a user-defined regular expression has a security concern, it is prohibited.
 
-      1. Return true.
+          1. Set |hasCondition| to true.
+      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/runningStatus}}"] [=map/exists=], then:
+          1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/runningStatus}}"] is neither of  {{RunningStatus/"running"}} nor {{RunningStatus/"not-running"}}, return false.
```

Comment:
I think this is prohibited by IDL. Since _rule_["`condition`"]["`runningStatus`"] is of type `RunningStatus`, it can never have a different value from those two.

Reply:
sure.  I have removed the check, and directly set |hasCondition| true.